samples/cpp/image_generation/image2image_concurrency.cpp

```diff
@@ -25,8 +25,28 @@ int32_t main(int32_t argc, char* argv[]) try {
     for (int32_t i = 2; i < argc - 1; ++i)
         prompts.push_back(argv[i]);
 
-    // Prepare initial pipeline and compiled models into device
-    pipelines.emplace_back(models_path, device);
+    ov::AnyMap properties;
+    if (device == "NPU") {
+        // Define static shape and guidance scale for NPU
+        const int num_images_per_prompt = 1;
+        const int height = 512;
+        const int width = 512;
+        const float guidance_scale = 7.5f;
+
+        pipelines.emplace_back(models_path);
+        pipelines.back().reshape(num_images_per_prompt, height, width, guidance_scale);
+        pipelines.back().compile(device);  // All models are compiled for NPU
+        //pipelines.back().compile("NPU", "NPU", "GPU");  // Compile for NPU and GPU, if needed
+
+        // Don't specify N, H, W, and guidance_scale in the properties map because they were made static
+        properties = ov::AnyMap{ov::genai::strength(0.8f),
+                                ov::genai::num_inference_steps(2)};
+    } else {
+        pipelines.emplace_back(models_path, device);
+
+        properties = ov::AnyMap{ov::genai::strength(0.8f),  // controls how initial image is noised after being converted to latent space. `1` means initial image is fully noised
+                                ov::genai::num_inference_steps(2)};
```

Comment:
`num_inference_steps` was 4 for that sample (and 2 for samples/cpp/image_generation/text2image_concurrency.cpp). I'd propose to make it 4 for samples/cpp/image_generation/image2image_concurrency.cpp. Unless @likholat says the opposite